docker-compose/db/db-init.sql

```diff
@@ -0,0 +1,6 @@
+-- Make sure any changes to these variables in the .env file are reflected here
+-- DB_DATABASE
+-- DB_USERNAME
+-- DB_PASSWORD
+GRANT ALL ON hacktober.* TO 'hacktober-user'@'%' IDENTIFIED BY 'password';
```

Comment:
I wasn't able to find a way to get environment variables into a SQL script - if you know of a way to do this I will un-hardcode these values.